.github/workflows/android.yml

```diff
@@ -46,3 +46,22 @@ jobs:
           name: debugApk
           path: |
             ./android/app/build/outputs/apk/debug/
+      - name: Patch nonfree
```

Comment:
Can you please separate this from the normal android job? Both free and non-free builds should be separated.
The new job would trigger the same way as the normal one.

Reply:
Done in 762eff2344b9b5db6ca7791729aaa5160564a2d3